.github/workflows/staging.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [13.x]
+        node-version: [14.x]
```

Comment:
Do this for all workflows please :-)

Reply:
Odd, for some reason main.yml doesn't have this piece in there, please add.

Reply:
One more to go :stuck_out_tongue_closed_eyes: https://github.com/hubmapconsortium/ccf-asct-reporter/blob/angular12Update/.github/workflows/build-previews.yml#L13